packages/gatsby-recipes/src/providers/npm/package.js

```diff
@@ -48,7 +61,7 @@ const executeInstalls = async root => {
   const depType = installs[0].resource.dependencyType
   const packagesToInstall = types[depType]
   installs = installs.filter(
-    i => !_.some(packagesToInstall, p => i.resource.id === p.resource.id)
+    i => !_.some(packagesToInstall, p => i.resource.name === p.resource.name)
```

Comment:
```suggestion
    i => !packagesToInstall.some(p => i.resource.name === p.resource.name)
```
Any reason to not use `Array.prototype.some`?

Reply:
nope!